django_s3_storage/storage.py

```diff
@@ -130,7 +130,7 @@ class S3Storage(Storage):
     """
 
     default_auth_settings = {
-        "AWS_REGION": "us-east-1",
+        "AWS_REGION": "eu-west-1",
```

Comment:
I don't think you meant to change this part...

```suggestion
        "AWS_REGION": "eu-east-1",
```

Reply:
Absolutely. This got in by accident.